community/kernel/src/main/java/org/neo4j/kernel/impl/api/index/OnlineIndexUpdatesValidator.java

```diff
@@ -55,11 +58,18 @@ public OnlineIndexUpdatesValidator( NeoStore neoStore, PropertyLoader propertyLo
     }
 
     @Override
-    public ValidatedIndexUpdates validate( TransactionRepresentation transaction )
-            throws IOException
+    public ValidatedIndexUpdates validate( TransactionRepresentation transaction ) throws IOException
     {
         NodePropertyCommandsExtractor extractor = new NodePropertyCommandsExtractor();
-        transaction.accept( extractor );
+        try
+        {
+            transaction.accept( extractor );
+        }
+        catch ( IOException cause )
```

Comment:
Is this really a panic? This is before we even start appending to log and committing, right?